shell/platform/darwin/graphics/BUILD.gn

```diff
@@ -0,0 +1,65 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+assert(is_ios)
```

Comment:
I plan on making this folder be compatible with macOS as well. Metal code will be usable mostly as-is. I will wire the `shell_enable_metal` flag to also support mac.